### PR TITLE
Thermal Comfort Mean Radiant Temperature Calculation Upgrades

### PR DESCRIPTION
**Pull request by RKStrand**
Pull request overview
---------------------
This pull request addresses issues that first came up in GitHub Issue #3469.  This started as a documentation fix and then became a more critical look at how we are calculating various flavors of MRT in the thermal comfort routine.  Two were identified as having something less than ideal.  The surface weighted option which averages one surface temperature and the zone MRT was basically double counting the surface that the person was closest to.  So, this was altered so that the zone MRT used for this calculation did not include the effect of the surface the person was closest to.  The angle factor option was using a simplified equation that did not account for surface emissivity and was also using a linear version of the equation.  Both of those deficiencies were addressed with this work.  Two unit tests and documentation mods were also completed.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue #3469 

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Code style (parentheses padding, variable names)
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add rules to spreadsheet
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Documentation changes in place
 - [ ] Changed docs build successfully
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

File: src/EnergyPlus/ThermalComfort.cc
Comment (mjwitte) [inline src/EnergyPlus/ThermalComfort.cc]: Nice efficiency idea to precalculate this, but `InsideAbsorpThermal` can be changed with EMS, so this needs to be recalculated every timestep just in case.

File: src/EnergyPlus/ThermalComfort.cc
Comment (mjwitte) [inline src/EnergyPlus/ThermalComfort.cc:1843]: This function does more than this, it gets the final answer including the weighted surface.

File: src/EnergyPlus/ThermalComfort.cc
Comment (mjwitte) [inline src/EnergyPlus/ThermalComfort.cc]: Don't want these warnings to write every timestep.  Sufficient to write this once - and add "only the first occurrence is reported" or something to that effect.

File: src/EnergyPlus/ThermalComfort.cc
Comment (mjwitte) [inline src/EnergyPlus/ThermalComfort.cc:1879]: Again, this can't be precalculated if EMS actuates the thermal absorptance.  Since this is acting on a single zone now, it should simple enough to just have one loop that calculates both ZoneAESum and SumAET, exlcuding the weighted surface.

**Review by mjwitte — COMMENTED** [review]
Some initial comments.

**Comment by mjwitte** [comment]
@RKStrand Also, I merged in latest develop on this branch, so be sure to pull before making changes.

File: src/EnergyPlus/ThermalComfort.cc
Comment (RKStrand) [reply src/EnergyPlus/ThermalComfort.cc]: @mjwitte Ah, I didn't realize that--good catch.  So much for efficiency when you have EMS!  I will fix this.

File: src/EnergyPlus/ThermalComfort.cc
Comment (RKStrand) [reply src/EnergyPlus/ThermalComfort.cc:1843]: @mjwitte Good point.  I will correct this.

File: src/EnergyPlus/ThermalComfort.cc
Comment (RKStrand) [reply src/EnergyPlus/ThermalComfort.cc:1879]: @mjwitte Got it.  Will fix this also.

File: src/EnergyPlus/ThermalComfort.cc
Comment (RKStrand) [reply src/EnergyPlus/ThermalComfort.cc]: @mjwitte Ok, I will change this as well.  It shouldn't ever happen, but I get that if it does that no one wants a 20Gb file with this message repeated over and over and over again.

**Comment by Myoldmopar** [comment]
@RKStrand as soon as you get his comments addressed I'll make one final pass over this and if all is well I'll get it merged in.

File: doc/engineering-reference/src/simulation-models-encyclopedic-reference-005/occupant-thermal-comfort.tex
Comment (mjwitte) [inline doc/engineering-reference/src/simulation-models-encyclopedic-reference-005/occupant-thermal-comfort.tex:1065]: Minor detail, the nomenclature table is missing A and epsilon.

**Comment by RKStrand** [comment]
@mjwitte @Myoldmopar Ok, I think I have addressed all of Mike's comments and have pushed another commit up.  Once the CI cycle completes I will take one more look at the results but I think this should be ready to go.  Thanks again for your help!

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
I'll let the debug builds run before merging, but this seems good now.

**Comment by Myoldmopar** [comment]
Fixed both the build warning, and the Linux build error (which was due to failing documentation build).  Not waiting for another pass of CI, this should be good.  Merging.

**Pull request merged**
